xbmc/filesystem/FavouritesDirectory.cpp

```diff
@@ -201,6 +201,8 @@ std::string CFavouritesDirectory::GetExecutePath(const CFileItem &item, const st
       execute = StringUtils::Format("PlayMedia(%s)", StringUtils::Paramify(item.GetVideoInfoTag()->m_strFileNameAndPath).c_str());
     else if (item.IsMusicDb() && item.HasMusicInfoTag())
       execute = StringUtils::Format("PlayMedia(%s)", StringUtils::Paramify(item.GetMusicInfoTag()->GetURL()).c_str());
+    else if (item.IsPicture() && item.HasPictureInfoTag())
```

Comment:
Do we need the check for `HasPictureInfoTag()`? Because you don't use it afterwards. In case of video and music items we need to check for the infotag's existance because we use it afterwards but that's not the case here.